index.html

```diff
@@ -0,0 +1,224 @@
+            ><img
+            src="assets/Normal Button/Tertiary/Disabled.png"
+            alt="Message"
+            /></a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <!-- header -->
+    <section class="header">
+      <div class="welcome">
+        <h1>
+          Hey There. I<span>&#39;</span>m Marwan
+        </h1>
+      </div>
+      <div class="job-desc">
+        <h2>I<span>&#39;</span>m a Software Developer</h2>
+      </div>
+      <div class="intro">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don<span>&#39;</span>t hestiate to contact me.</p>
+      </div>
+      <div class="icons-container">
+        <ul>
+          <li><a><img src="assets/github.png" alt="github"/></a></li>
+          <li><a><img src="assets/linkedin.png" alt="linkedin"/></a></li>
+          <li><a><img src="assets/angel list.png" alt="angel-list"/></a></li>
+          <li><a><img src="assets/twitter.png" alt="twitter"/></a></li>
+          <li><a><img src="assets/medium.png" alt="medium"/></a></li>
+        </ul>
+        </div>
+    </section>  
+      <!-- Recent-Works -->
+    <section class="recent-works">
+
+        <h1>My Recent Work</h1>
+      <hr>  
+      <div class="head-work">
+        <img src="assets/Img Placeholder.png" alt="project-cover">
+        <h2>Multi-Post Stories</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>css</h3>
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+    </section>
+
+    <section class="works">
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+
```

Comment:
- Kindly make sure that you add an appropriate hover to your button, make the button (e.g., "Get my resume") a `link `a or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design) also do that for  The button (e.g., "See project"),  it will make your design interactive  to the Figma design as required in the project  :+1: